Doc/Core#666 - Move GlobalAppConfig attributes to CoreSection (#511)
Avaiga/taipy-doc: docs/manuals/core/config/core-config.md

> The `CoreSection^` holds configuration fields related to the Core application.

Reviewer: ```suggestion
The `CoreSection^` holds configuration fields related to the Core service.
```

Here are the (optional) configurable properties:

- _**root_folder**_: The path of the base folder for the Taipy application, its default value is "./taipy/".
- _**storage_folder**_: The folder name used to store Taipy data, its default value is ".data/". It is used in
  conjunction with the root_folder field. That means the default storage path is "./taipy/.data/".

> - _**read_entity_retry**_: The integer number of retry when reading an entity after a failed attempt (in case of

Reviewer: ```suggestion
- _**read_entity_retry**_: An integer number only used with `filesystem` _**repository_type**_. <br>
It corresponds to the number of times Taipy retries reading an entity after a failed attempt for concurrent access. <br>
The default value is 3.
```

>   concurrent access). The default value is 3.

Reviewer: ```suggestion
```

  If _**repository_type**_ is set to `filesystem`, Taipy uses _**root_folder**_ and _**storage_folder**_ to store
  entities. In this case, _**repository_properties**_ attribute is not required.</br>
  If _**repository_type**_ is set to `sql`, the _**db_location**_ attribute is required as the path of a sqlite3
  database file. Please refer to [SQL storage section](core-config.md#sql-storage-for-taipy-entities).</br>
  If _**repository_type**_ is set to `mongo`, the possible properties are _**mongodb_hostname**_, _**mongodb_user**_,
  _**mongodb_password**_, _**mongodb_port**_, _**application_db**_, _**properties**_. Please refer to
  [MongoDB storage section](core-config.md#mongodb-storage-for-taipy-entities).
- _**mode**_: A string that indicates the mode of the version management system.
  Possible values are *"development"*, *"experiment"*, or *"production"*.
- _**version_number**_: The identifier of the version. In development mode, the version number is ignored.
- _**force**_: Indicates whether Taipy will override a version even if the configuration has changed
    or not and run the application. Default to False.

> - _**clean_entities**_: Indicates whether running a Taipy Core service will clean all current version
>     entities or not before running the application. Default to False.

Reviewer: Sorry, I missed that in the other PRs for the ticket [core-#666](https://github.com/Avaiga/taipy-core/issues/666). Probably because it s a Demoniac ticket. 🤣

In the ticket, the field `clean_entities` should be completely removed from the config.
Since we have the development mode that does it automatically, and since we have the CLI to manage versions, it is not needed anymore. Plus, it is error-prone and can be dangerous to remove all production entities just with a config value.
I discussed with @FlorianJacta that agrees with that.

That means we need to remove the attribute from the `CoreSection`, and it also means we need to remove lines 205, 206, and 207 in `taipy-core/src/taipy/core/taipy.py`

```suggestion
```

Author: We need to update some example, and the code on taipy-core for this also. Do you want to do it in another ticket?

Reviewer: I don't mind. It s up to you.

Author: Ticket created: https://github.com/Avaiga/taipy-core/issues/693
